apps/ui/env.mjs

```diff
@@ -24,6 +24,7 @@ const ServerSchema = z.object({
   // mandatory because the SSR can't connect to the API without it
   // frontend SSR is running on a different host than the backend
   TILAVARAUS_API_URL: z.string().url(),
+  EMAIL_VARAAMO_EXT_LINK: z.string().url(),
```

Comment:
I'd put this to optional and default it to undefined, check for undefined (or empty string) when constructing the links. Has to be empty string or `null` because of json.

Reply:
Fixed